src/edu/stanford/nlp/sempre/Master.java

```diff
@@ -203,10 +234,14 @@ public synchronized Response processQuery(Session session, String line) {
   }
 
   String summaryString(Response response) {
+    try {
     if (response.getExample() != null)
       return response.getFormulaAnswer();
     if (response.getLines().size() > 0)
       return response.getLines().get(0);
+    } catch (Exception e) {
+      return null;
```

Comment:
Should return e.toString() or something?  Silent failures are sometimes no fun to track down...